cmd/config-v16.go

```diff
@@ -131,15 +131,15 @@ func loadConfig(envParams envParams) error {
 
 	// If env is set override the credentials from config file.
 	if globalIsEnvCreds {
-		srvCfg.SetCredential(envParams.creds)
+		srvCfg.SetCredential(globalActiveCred)
+	} else {
+		globalActiveCred = srvCfg.GetCredential()
```

Comment:
This makes a little confusion about whether we should use globalActiveCred or srvCfg.GetCredential() later. I think we should reduce global variables instead if possible.